src/DetailsView/Styles/detailsview.scss

```diff
@@ -636,15 +643,18 @@ div.insights-file-issue-details-dialog-container {
             padding-left: calc(0.5vw + 13px); // 13px is Chevron width.
         }
         .left-nav {
-            width: 200px;
+            max-height: calc(100vh - (#{$detailsViewTotalHeaderHeight}));
+            height: calc(100vh - (#{$detailsViewTotalHeaderHeight}));
+            overflow-y: auto;
+            width: calc($detailsViewLeftNavWidth - 1px); // border
```

Comment:
Should the variable value here be interpolated here?
`#{$detailsViewLeftNavWidth}` instead of `$detailsViewLeftNavWidth`
In Chrome and Edge DevTools, an error is reported:
![image](https://user-images.githubusercontent.com/1752950/66004684-3cd28880-e45e-11e9-9b45-bcebfd3b6e9e.png)